runtime/storage_test.go

```diff
@@ -8207,3 +9143,35 @@ func checkAccountStorageMapData(
 	require.Equal(tb, 1, len(rootSlabIDs))
 	require.Contains(tb, rootSlabIDs, accountSlabID)
 }
+
+func newSlabStorage(ledger atree.Ledger) *atree.PersistentSlabStorage {
```

Comment:
This looks like a part of `interpreter.NewStorage`. Maybe we can have this as `interpreter.NewPersistentSlabStorage` and use it in `interpreter.NewStorage` and in the tests here